http/alby_http_service.go

```diff
@@ -29,6 +29,7 @@ func NewAlbyHttpService(svc service.Service, albyOAuthSvc alby.AlbyOAuthService,
 func (albyHttpSvc *AlbyHttpService) RegisterSharedRoutes(restrictedGroup *echo.Group, e *echo.Echo) {
 	e.GET("/api/alby/callback", albyHttpSvc.albyCallbackHandler)
 	e.GET("/api/alby/info", albyHttpSvc.albyInfoHandler)
+	e.GET("/api/alby/btc_rate", albyHttpSvc.albyBitcoinRateHandler)
```

Comment:
can you please try to be consistent with the other routes? why `btc_rate`?

Reply:
i changed endpoint name to rates